creditcoin-js/README.md

```diff
@@ -1,6 +1,125 @@
 # creditcoin-js
 
-**WARNING**: This is an alpha version of creditcoin-js. It is not ready for production use. It will have breaking changes often.
+## Getting started
+
+### Preqrequisites
+
+Creditcoin-js requires the following to be installed:
+
+-   [Node.js](https://nodejs.org/en/)
+-   [TypeScript](https://www.typescriptlang.org/)
+
+### Install
+
+Adding Creditcoin-JS to your project is easy. Install it by using your favorite package manager:
+
+```shell
+yarn add creditcoin-js
+```
+
+This will install the latest release version, which should allow you to interact with Creditcoin's main network and your own local chains that use the latest Creditcoin binaries.
+
+## Usage
+
+### Import
+
+Importing the library into your project:
+
+```typescript
+import { creditcoinApi } from 'creditcoin-js';
+
+const { api } = await CreditcoinApi('ws://localhost:9944');
+```
+
+### Using the API
+
+The API is a collection of modules that provide access to the various functions of the Creditcoin blockchain.
+
+```typescript
+const { api, extrinsics, utils } = await CreditcoinApi('ws://localhost:9944');
+```
+
+### Creating transactions
+
+```typescript
+const { api } = await CreditcoinApi('ws://localhost:9944');
+
+const tx = api.
+    .tx
+    .balances
+    .transfer(
+        "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+        "1000000000000000"  // CTC amount in microunits
+                            // (1 CTC = 1e18 microunits)
+    )
+```
+
+### Signing & sending
+
+```typescript
+import { Keyring } from 'creditcoin-js';
+
+const keyring = new Keyring({ type: 'sr25519' });
+const alice = keyring.addFromUri('//Alice');
```

Comment:
Probably better to also add a commented out line about creating a keyring from a private key b/c that's what most users will be doing.